src/objects/camera.ts

```diff
@@ -22,11 +22,10 @@ export class Camera extends SpaceEntity {
 	near: number;
 	far: number;
 
-	constructor(config: CameraConfig = {}, position: Vector3 = Vector3.zero, direction: Vector3 = Vector3.backward) {
+	constructor(config: CameraConfig = {}, position: Vector3 = Vector3.zero) {
 		super();
 
 		this.setPosition(position);
-		this.setDirection(direction);
```

Comment:
@Tucchhaa Why was this removed?

Reply:
Setting direction for the camera is unstable. Remember direction doesn't have Z rotation. It's better to use camera.rotate